src/app/teamperiods/teamperiods.component.html

```diff
@@ -1,14 +1,24 @@
 <mat-spinner *ngIf="!isLoaded()"></mat-spinner>
 <div *ngIf="isLoaded()" class="grid-container">
-<mat-card>
-<mat-card-header>
-  <mat-card-title (click)="editTeam()">{{team?.displayName}}: Periods</mat-card-title>
-</mat-card-header>
-<mat-card-content>
-<mat-list>
-  <mat-list-item *ngFor="let period of sortedPeriods()"><a routerLink="/team/{{team!.id}}/period/{{period!.id}}">{{period?.displayName}}</a></mat-list-item>
-</mat-list>
-<button mat-raised-button [disabled]="!userHasEditPermissions" (click)="addPeriod()" color="primary">Add Period</button>
-</mat-card-content>
-</mat-card>
+  <mat-card>
+    <mat-card-header>
+      <mat-card-title>
+        {{team?.displayName}}: Periods
+        <button type="button" mat-icon-button aria-label="Edit Team" (click)="editTeam()">
+          <mat-icon>edit</mat-icon>
+        </button>
+      </mat-card-title>
+    </mat-card-header>
+    <mat-card-content>
+      <mat-nav-list>
+        <a mat-list-item *ngFor="let period of sortedPeriods()" routerLink="/team/{{team!.id}}/period/{{period!.id}}">{{period?.displayName}}</a>
+      </mat-nav-list>
+    </mat-card-content>
+    <mat-card-actions>
+      <a routerLink="/teams" aria-label="Go back teams page" mat-icon-button>
```

Comment:
Nit: it would be nice to make the `aria-label` a complete sentence, i.e. "Go back *to the* teams page".